amical/tests/test_extraction.py

```diff
@@ -7,6 +7,11 @@
 from amical.mf_pipeline.bispect import _add_infos_header
 
 
+# Astropy versions for
+ASTROPY_VERSION = Version(astropy.__version__)
+ASTROPY_WORKING = Version("5.0rc")
```

Comment:
I'd suggest not using a global const for this one. For now we only have one place where we adapt behaviour as a function of satrapy's version, but in the future there could be more, in which case this variable name will become unclear and confusing.

Reply:
Fixed by the last commit. 5.0rc is used everywhere directly now.